src/System Application/App/Page Style/PageStyle.al

```diff
@@ -0,0 +1,49 @@
+enum 50000 "Page Style"
```

Comment:
I think the name "Page Style" is not perfekt. Its about Labels and Fields. Maybe "Control Style Type"  would be better.

Reply:
I tend to agree, that the styling is for controls, rather than a page object. "Control Style" sounds good to me.

Reply:
renamed to "Control Style"
set Namespace System.Visualization